src/main/scala/org/broadinstitute/dsde/workbench/leonardo/db/ClusterComponent.scala

```diff
@@ -295,6 +303,21 @@ trait ClusterComponent extends LeoComponent {
         .result map { recs => unmarshalFullCluster(recs)}
     }
 
+    // TODO: find how to make these not full cluster queries
```

Comment:
+1, definitely avoid `fullClusterQuery` if you can. I'm learning in perf testing that it has some performance issues...